vernac/g_vernac.mlg

```diff
@@ -581,6 +581,12 @@ GRAMMAR EXTEND Gram
       | IDENT "Collection"; id = identref; ":="; expr = section_subset_expr ->
           { VernacNameSectionHypSet (id, expr) }
 
+      (* Requiring an external file *)
+      | IDENT "From" ; ns = global ;
```

Comment:
Would it be helpful to make the "From" optional, in which case the filename would be relative to the directory of the current file?